openassessment/data.py

```diff
@@ -670,3 +676,178 @@ def collect_ora2_responses(cls, course_id, desired_statuses=None):
                 result[item_id][status] += 1
 
         return result
+
+
+class OraDownloadData:
```

Comment:
I don't really like that this is a class that only has classmethods. That seems to indicate that these are better off as simple functions in `data.py`. But I see that you're copying the pattern that was here already, in `OraAggregateData`. So I suppose it's ok for now.

Reply:
>as simple functions in data.py

I agree. Probably, since there are already two such classes, we can split `data.py` on three different files, like `aggregate.py`, `download.py` and `utils.py` (for `CsvWriter`).